File: APIs/vulkan 2022/14 - Push Constants/Finished/scene.cpp
#include "scene.h"

Scene::Scene() {

	for (float x = -1.0f; x < 1.0f; x += 0.2f) {

		for (float y = -1.0f; y < 1.0f; y += 0.2f) {

			trianglePositions.push_back(glm::vec3(x, y, 0.0f));

		}

	}

};

File: APIs/pyvulkan/14 - Push Constants/finished/scene.py
from config import *

class Scene:


    def __init__(self):

        self.triangle_positions = []

        x = -1.0
        while x < 1.0:
            y = -1.0
            while y < 1.0:
                self.triangle_positions.append(
                    np.array([x, y, 0], dtype = np.float32)
                )
                y += 0.2
            x += 0.2